.github/workflows/milestone-checker.yml

```diff
@@ -15,8 +15,9 @@ on:
 jobs:
   # checks that a milestone entry is present for a PR
   milestone-check:
-    # If there is a `pr/no-milestone` label we ignore this check
-    if: "!contains(github.event.pull_request.labels.*.name, 'pr/no-milestone')"
+    # If there is a `pr/no-milestone` label, or this comes from a fork (community contributor) we ignore this check
+    if: ${{ (github.repository == 'hashicorp/vault' && (github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name))
```

Comment:
I'm not 100% sure if this syntax works for multi-line `if`? The only examples I see in our code base uses `if: |`, and I don't see anything in the GitHub docs.

Reply:
It's how we have things for `security-scan.yml`, and it seems to work there, for what it's worth

Reply:
Ah, cool. LGTM then.